scss/common/_forms.scss

```diff
@@ -225,4 +225,89 @@
         right: 3px;
     }
 
+}
+
+@include exports("forms/integration-with-bootstrap-forms") {
+    $bootstrap-integration-spacer: 1rem !default;
+
+    .k-widget.form-control {
+        padding: 0;
+    }
+
+    .k-widget.form-control,
+    .k-maskedtextbox.form-control .k-textbox,
+    .k-textbox.form-control {
+        width: 100%;
+    }
+
+    .k-autocomplete.form-control .k-input {
+        box-sizing: border-box;
+    }
+
+    .input-group {
+        .k-dateinput-wrap,
+        .k-dropdown-wrap,
+        .k-multiselect-wrap,
+        .k-numeric-wrap,
+        .k-picker-wrap,
+        .k-maskedtextbox .k-textbox {
+            border-top-right-radius: 0;
+            border-bottom-right-radius: 0;
+
+            .k-select {
+                border-top-right-radius: 0;
+                border-bottom-right-radius: 0;
+            }
+        }
+
+        .k-widget.form-control:not(:first-child):not(:last-child) {
+            .k-dateinput-wrap,
+            .k-dropdown-wrap,
+            .k-multiselect-wrap,
+            .k-numeric-wrap,
+            .k-picker-wrap,
+            > .k-textbox {
+                border-radius: 0;
+            }
+        }
+    }
+
+    .input-group-addon + .k-widget.form-control {
+        .k-dateinput-wrap,
+        .k-dropdown-wrap,
+        .k-multiselect-wrap,
+        .k-numeric-wrap,
+        .k-picker-wrap,
+        > .k-textbox {
+            border-top-left-radius: 0;
+            border-bottom-left-radius: 0;
+            border-top-right-radius: $border-radius;
+            border-bottom-right-radius: $border-radius;
+        }
+    }
+
+    .k-widget.form-control .k-input,
+    .k-maskedtextbox.form-control .k-textbox,
+    .k-textbox.form-control {
+        padding: ($bootstrap-integration-spacer / 2) ($bootstrap-integration-spacer * .75);
+    }
+
+    $bootstrap-integration-input-height: $bootstrap-integration-spacer * 2.25 !default;
```

Comment:
It appears that the fractions are based on the values available in the Bootstrap framework (`2.25`, for example, seems oddly specific). How is this going to be used in `kendo-theme-bootstrap`?